examples/curl/run.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+set -e
```

Comment:
My usual default is `set -euo pipefail`, which catches a few other footguns

Reply:
Thanks, changed it to this.